gallery/gallery/lib/studies/crane/header_form.dart

```diff
@@ -75,7 +75,7 @@ class _HeaderTextField extends StatelessWidget {
         fillColor: cranePurple700,
         filled: true,
         hintText: field.title,
-        hasFloatingPlaceholder: false,
+        floatingLabelBehavior: FloatingLabelBehavior.never,
```

Comment:
What is this change for?

Reply:
Removes a deprecated property. It's not in stable yet so I reverted it